src/os/unix/mod.rs

```diff
@@ -19,26 +19,43 @@ pub use x11::Menu;
 
 use std::os::raw;
 
-//Differentiate between Wayland and X11 at run-time
+// Differentiate between Wayland and X11 at run-time
 pub enum Window {
     X11(x11::Window),
-    Wayland(()), //WlWindow
+    Wayland(wayland::Window),
```

Comment:
And these.

Should probably also make this nonexhaustive by adding something like:

```rust
#[doc(hidden)]
__Nonexhaustive,
```

Reply:
Why do you think this is important?

Reply:
My reasoning was: to force consumers to handle all cases. Now, I can see that it isn't important.